### DISCUSSION TITLE
update commit in github

### DISCUSSION THREAD

**Discussion by immaculeebyaruhanga** [community/community#109543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

my commits remain static while the contributions increase every day, I even made some configurations with my email which is on github but I don't know if what the problem is I would like you to help me to solve it in such a way that the commits increase each time I push the branches

**Comment by NipulM** [comment]
Hey @immaculeebyaruhanga! 👋🏼

If you're having trouble with your Git settings, it might be worth double-checking your credentials:

Make sure your Git configurations match the details on your GitHub account. You can do this by running:
```
git config user.name
git config user.email
```

If they don't match up, you can update them using:
```
git config --global user.name "Your Name"
git config --global user.email "your.email@example.com"
```

Hope this helps!

**Reply by immaculeebyaruhanga** [reply]
i would like to do fusion of all commit wich is in different repositories and wich is in readme in brief all contibutions

**Reply by immaculeebyaruhanga** [reply]
![image](https://github.com/community/community/assets/155079185/29dc5940-588e-4ea2-be7d-0dac34330f9d)
this i can 't understand my wish is to fusion all commit
